### ISSUE TITLE
Touch Target of Clickable Text

### ISSUE THREAD

**Issue by leancoelho** [jonataslaw/readmore#67]
Is there anyway of increasing the touch area of the 'Read more' or 'Show more' text? 
For accessibility purpose i need to increase this touch target to at least 48dx, i could increase the text font size, but it should be possible to only increase the touch area of the clickable text.
